CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ set(MLIR_MIOPEN_DRIVER_TEST_GPU_VALIDATION 1 CACHE BOOL "Enable E2E tests with G
 
 set(MLIR_MIOPEN_SQLITE_ENABLED 0 CACHE BOOL "Enable SQLite integration")
 set(MLIR_ENABLE_SQLITE "${MLIR_MIOPEN_SQLITE_ENABLED}")
+set(LLVM_ENABLE_ZLIB "OFF" CACHE STRING "")
```

Comment:
```suggestion
set(LLVM_ENABLE_ZLIB OFF CACHE BOOL "")
```

Reply:
LLVM_ENABLE_ZLIB is a string variable.

external/llvm-project/llvm/CMakeLists.txt:

set(LLVM_ENABLE_ZLIB "ON" CACHE STRING "Use zlib for compression/decompression if available. Can be ON, OFF, or FORCE_ON")

Reply:
Nvm